delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/disk/DiskStorageAccess.scala

```diff
@@ -0,0 +1,25 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.disk
+
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.EncryptionState.Decrypted
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageRejection.StorageNotAccessible
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageValue.DiskStorageValue
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.StorageAccess
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import monix.bio.IO
+
+import java.nio.file.Files
+
+object DiskStorageAccess extends StorageAccess {
+  override type Storage = DiskStorageValue[Decrypted]
+
+  override def apply(id: Iri, storage: DiskStorageValue[Decrypted]): IO[StorageNotAccessible, Unit] = {
```

Comment:
I would recommend delaying the IO operations because they are side effecting. Things like retry won't work.